### ISSUE TITLE
Lighthouse accessibility doesn't like our viewport meta tag

### ISSUE THREAD

**Issue by kevmoo** [flutter/flutter#122127]
```html
<meta flt-viewport="" name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```

![image](https://user-images.githubusercontent.com/17034/223543616-b72cb2a9-d9f4-4c26-ad91-82012da3ee25.png)

**Comment by ditman** [comment]
This was there as a way of preventing pinch-zoom in mobile apps. We use `touch-action: none` in [FullScreen mode](https://github.com/flutter/engine/blob/237ab63bfbe340e7ce961a0735774c11521cb240/lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/full_page_embedding_strategy.dart#L80), so the fix for this would be to find the value that lighthouse likes that works for Flutter Web :)

(Note that `touch-action: none` is Safari > 13, so older versions of the browser may let users pinch-zoom)

**Comment by kevmoo** [comment]
Dropping old Safari might be doable: https://caniuse.com/usage-table

**Comment by ditman** [comment]
Yes, currently we're using Safari 13 as our browser baseline, but we could even push that forward "because all devices that supported iOS 13 support iOS 14 and iOS 15 as well."

**Comment by ditman** [comment]
(`width=device-width, initial-scale=1` may be Just Enough for what we need)

**Comment by yjbanov** [comment]
Dropping to P4 for now, since there's no customer request for this yet.

**Comment by 0ttik** [comment]
There is a customer request for it. Especially annoying it is on Android when working with text. For that cases behaviour is very unpleasent.

https://github.com/flutter/flutter/issues/97305

**Comment by flutter-zl** [comment]
this issue was fixed by https://github.com/flutter/flutter/pull/182047.
